combat-trainer.lic

```diff
@@ -3449,6 +3472,27 @@ class GameState
     true
   end
 
+  def update_analyze_array_combo
+    ret = false
+    if Flags["ct-accuracy-ready"] && @barb_accuracy
+      perform_analyze?('accuracy', 50)
```

Comment:
❓ Should this do something with the return value of `perform_analyze?`?

Reply:
This sends the perform_analyze values in update_analyze_array_combo instead of update_analyze_array, just doing each analyze specifically rather than shifting to each one